wsgi.py

```diff
@@ -1,3 +1,5 @@
+import sys
+sys.path.append('/var/www/indexd')
```

Comment:
Not sure it matters, but what was the reason for adding this to the path here rather than adding it in the Apache config?